proxy/proxy/client.go

```diff
@@ -73,7 +75,7 @@ type Client struct {
 	// instances. This value is defined by the server-side code, but for now it
 	// should always be 3307.
 	Port int
-	// Required; specifies how certificates are obtained.
+	// Certs specifies how certificates are obtained.
```

Comment:
Why did we drop `Required` here?

Reply:
Go docs should start with the thing they're documenting, but since it's unrelated to the PR here, I'll just restore the original and we can address this some other time.